CHANGE_LOG.md

```diff
@@ -38,6 +38,8 @@ All notable changes to this project will be documented in this file.
 - Added public/index.html template with proper styling and configuration guidance for TypeScript sample
 - Added .gitignore file to exclude build artifacts and dependencies from TypeScript sample
 - Added browserslist configuration to TypeScript sample package.json for better browser compatibility
+- Add QueueOverflowHandleMiddleware to display disconnection banner if conversation is ended due to overflow
```

Comment:
Left over from another PR that must have been in the same branch at least briefly. Please delete this line and let the other PR add it.

Reply:
Never mind, you are just changing the changelog to make sure the correct h3 has the change message. It is added vs. Changed. Got it. Resolve this one.